task/prefetch-dependencies/0.1/prefetch-dependencies.yaml

```diff
@@ -82,164 +82,13 @@ spec:
         yq 'del(.goproxy_url)' <<< "${CONFIG_FILE_CONTENT}" > /mnt/config/config.yaml
       fi
 
-  - name: check-prefetch-input
-    image: quay.io/konflux-ci/cachi2:0.19.0@sha256:f67e7f9771c3d425bdd917313553c882d0ed942076dc31d9ce35524bdda1fb6e
-    # per https://kubernetes.io/docs/concepts/containers/images/#imagepullpolicy-defaulting
-    # the cluster will set imagePullPolicy to IfNotPresent
-    env:
-    - name: INPUT
-      value: $(params.input)
-    script: |
-      if [ -z "${INPUT}" ]
-      then
-        # Confirm input was provided though it's likely the whole task would be skipped if it wasn't
-        echo "No prefetch will be performed because no input was provided for cachi2 fetch-deps"
-        echo "skip" > /shared/skip
-      fi
```

Comment:
I think we need to replicate this skip logic somewhere

Reply:
Sigh, great catch! I reverted this bit of logic prior to commit 5f62fe3b01edcd6897a00c1c156d4f7272a0e31e .